Template/LO/Source/setrun.f

```diff
@@ -15,6 +15,7 @@ subroutine setrun
       include 'PDF/pdf.inc'
       include 'run.inc'
       include 'alfas.inc'
+      include 'vector.inc'
       include 'MODEL/coupl.inc'
```

Comment:
Do no see why this is needed (at least from the diff of the file)
-> ok got it because of coupl.inc 
Can we add a comment about it next to the include?

Reply:
Yes here vector.inc was included because of coupl.inc. I have added the comment inside coupl.inc actually. In generated code it looks like this
https://github.com/madgraph5/madgraph4gpu/blob/79a0b8f71f3a9e2922494493250c5fd8985706c0/epochX/cudacpp/gg_tt.mad/Source/MODEL/coupl.inc#L7
```
C     NB: VECSIZE_MEMMAX is defined in vector.inc
C     NB: vector.inc must be included before coupl.inc
C     

      DOUBLE PRECISION G, ALL_G(VECSIZE_MEMMAX)
      COMMON/STRONG/ G, ALL_G
```

But I agree that I can add extra comments. How about changing the various
```
      include 'vector.inc'
```
into
```
      include 'vector.inc' ! needed by coupl.inc (defines VECSIZE_MEMMAX)
```
Looks better?

Reply:
yes wonderfull. Thanks a lot

Reply:
Done here https://github.com/mg5amcnlo/mg5amcnlo/pull/23/commits/49100837bd872e4f4abe104ddb0c1a974939741f

Can be resolved